client/my-sites/upgrades/domain-search/site-redirect.jsx

```diff
@@ -35,7 +37,7 @@ var SiteRedirect = React.createClass( {
 	checkSiteIsUpgradeable: function( ) {
 		var selectedSite = this.props.sites.getSelectedSite();
```

Comment:
This looks like the perfect place to migrate this to use the `getSelectedSite` selector, once you change the usage in `mapStateToProps` to use it instead of `sitesList`.